telemetry/definitions/commonDefinitions.json

```diff
@@ -1260,6 +1260,11 @@
             "type": "string",
             "description": "User locale. Examples: en-US, en-GB, etc."
         },
+        {
+            "name": "module",
+            "type": "string",
+            "description": "A generic module metadata"
```

Comment:
Example values will help devs understand how/when to use this

Reply:
Updated in https://github.com/aws/aws-toolkit-common/pull/779